staging/src/k8s.io/apiserver/pkg/util/flowcontrol/controller_test.go

```diff
@@ -139,7 +140,7 @@ func (cqs *ctlrTestQueueSet) IsIdle() bool {
 	return cqs.countActive == 0
 }
 
-func (cqs *ctlrTestQueueSet) StartRequest(ctx context.Context, width uint, hashValue uint64, flowDistinguisher, fsName string, descr1, descr2 interface{}, queueNoteFn fq.QueueNoteFn) (req fq.Request, idle bool) {
+func (cqs *ctlrTestQueueSet) StartRequest(ctx context.Context, width fcrequest.Width, hashValue uint64, flowDistinguisher, fsName string, descr1, descr2 interface{}, queueNoteFn fq.QueueNoteFn) (req fq.Request, idle bool) {
```

Comment:
nit: we probably should be passing a pointer instead of whole struct in all those calls, but we can do that in a follow up
@tkashem

Reply:
using pointer now, (since I had to force push to fix the test anyway)